examples/route-mirroring/front-envoy.yaml

```diff
@@ -0,0 +1,105 @@
+static_resources:
+  listeners:
+  - address:
+      socket_address:
+        address: 0.0.0.0
+        port_value: 8080
+    filter_chains:
+    - filters:
+      - name: envoy.filters.network.http_connection_manager
+        typed_config:
+          "@type": type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
+          codec_type: AUTO
+          stat_prefix: ingress_http
+          route_config:
+            name: local_route
+            virtual_hosts:
+            - name: backend
+              domains:
+              - "*"
+              routes:
+              - match:
+                  prefix: "/service/1"
+                route:
+                  cluster: service1
+                  request_mirror_policies:
+                    cluster: "service1-mirror"
+              - match:
+                  prefix: "/service/2"
+                route:
+                  cluster: service2
+                  request_mirror_policies:
+                    cluster_header: "x-mirror-cluster"
```

Comment:
is the `x-mirror-cluster` specific header? or we can name it with other names?

Reply:
@nguyenptk the header can be anything - but that's what the request must use to select the mirror cluster